processing/src/main/java/io/druid/segment/incremental/IncrementalIndexStorageAdapter.java

```diff
@@ -179,8 +180,13 @@ public DateTime getMaxIngestedEventTime()
 
     final Interval actualInterval = actualIntervalTmp;
 
+    Iterable<Long> iterable = gran.iterable(actualInterval.getStartMillis(), actualInterval.getEndMillis());
+    if (descending) {
+      // todo move into granularity
```

Comment:
can we use uppercase TODO so it gets highlighted, or just make it a comment saying this would be better done in granularity